x-pack/plugins/apm/typings/es_schemas/ui/fields/agent.ts

```diff
@@ -29,6 +27,12 @@ export type AgentName =
   | 'opentelemetry/ruby'
   | 'opentelemetry/webjs';
 
+/*
+ * Support additional agent types by appending definitions in mappings.json
+ * (for telemetry) and the AgentName type.
+ */
+export type AgentName = ElasticAgentName | OpenTelemetryAgentName;
```

Comment:
```suggestion
export type AgentName = 
  | 'go'
  | 'java'
  | 'js-base'
  | 'rum-js'
  | 'nodejs'
  | 'python'
  | 'dotnet'
  | 'ruby'
  | 'otlp'
  | 'opentelemetry/cpp'
  | 'opentelemetry/dotnet'
  | 'opentelemetry/erlang'
  | 'opentelemetry/go'
  | 'opentelemetry/java'
  | 'opentelemetry/nodejs'
  | 'opentelemetry/php'
  | 'opentelemetry/python'
  | 'opentelemetry/ruby'
  | 'opentelemetry/webjs';
```

Reply:
I've created https://github.com/elastic/kibana/pull/79654 as a fix to the `@kbn/telemetry-tools` 😉